dev/source/docs/static-analysis.rst

```diff
@@ -0,0 +1,41 @@
+.. _static-analysis:
+
+===============
+Static Analysis
+===============
+
+Two tools are often used by ArduPilot to perform static analysis on the codebase, clang's "scan build" option and Coverity.
```

Comment:
Coverity does not seem to be shown

Reply:
Coverity is a bit tricky as it is done with a cron job on travis